src/classes/HiFiMixerSession.ts

```diff
@@ -252,7 +261,7 @@ export class HiFiMixerSession {
             let initTimeout = setTimeout(async () => {
                 let errMsg = `Couldn't connect to mixer: Call to \`init\` timed out!`
                 try {
-                    await this.disconnectFromHiFiMixer();
+                    await this.disconnectFromHiFiMixer(true);
```

Comment:
There are several of these. See [L597](https://github.com/highfidelity/hifi-spatial-audio-js/pull/128/files#diff-119c1e67189140964dcdf8544a54d44ab7517ac1354427b54431056b718bf94bR597), below